### PR TITLE
close leftpane overlay on message sent to that conversation

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: ts/components/session/LeftPaneContactSection.tsx [hunk 1]
```diff
@@ -90,6 +85,8 @@ export class LeftPaneContactSection extends React.Component<Props, State> {
 
   public componentDidMount() {
     MainViewController.renderMessageView();
+
+    window.Whisper.events.on('calculatingPoW', this.closeOverlay);
   }
 
   public componentDidUpdate() {
```
Comment (Mikunj) [inline ts/components/session/LeftPaneContactSection.tsx:92]: for readability sake, can we move `componentDidMount` and `componentDidUpdate` above `componentWillUnmount`?

File: ts/components/session/LeftPaneMessageSection.tsx [hunk 2]
```diff
@@ -178,12 +180,19 @@ export class LeftPaneMessageSection extends React.Component<Props, State> {
 
   public componentDidMount() {
```
Comment (Mikunj) [inline ts/components/session/LeftPaneMessageSection.tsx:181]: Same here

Comment (Bilb) [reply ts/components/session/LeftPaneMessageSection.tsx:181]: fixed

File: ts/components/session/LeftPaneContactSection.tsx (hunk 1 above)
Comment (Bilb) [reply ts/components/session/LeftPaneContactSection.tsx:92]: fixed

**Review by Mikunj — APPROVED** [review]

**Pull request merged**